src/components/graph/graph.config.js

```diff
@@ -266,6 +270,8 @@ export default {
         renderLabel: false,
         semanticStrokeWidth: false,
         strokeWidth: 1.5,
+        markerHeight: "6",
```

Comment:
These values of `markerHeight` and `markerWidth` should be numbers ([source mdn docs](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/markerHeight#Usage_notes))